packages/edit-site/src/components/header/main-dashboard-button/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import FullscreenModeClose from '../fullscreen-mode-close';
+
+const { Fill: MainDashboardButton, Slot } = createSlotFill(
+	'SiteEditorMainDashboardButton'
+);
+
+MainDashboardButton.Slot = () => (
+	<Slot>
+		{ ( fills ) => {
+			// Return default Close button if no fills are provided, otherwise replace it with available fills.
+			if ( isEmpty( fills ) ) {
+				return <FullscreenModeClose />;
+			}
+
+			return <> { fills } </>;
```

Comment:
These spaces are significant, and cause React to create two additional text node children.

https://codepen.io/aduth/pen/ZEbrOYY

```suggestion
			return <>{ fills }</>;
```

Reply:
Good catch

Reply:
Thanks for pointing it out! I removed this code in https://github.com/WordPress/gutenberg/pull/22179.